timed/tracking/tests/test_report.py

```diff
@@ -93,8 +94,10 @@ def test_report_intersection_full(auth_client):
 
 def test_report_intersection_partial(auth_client):
     user = auth_client.user
-    ReportFactory.create(review=True, not_billable=True, comment="test")
+    report = ReportFactory.create(review=True, not_billable=True, comment="test")
```

Comment:
Would you mind using the `report_factory` fixture for this and do it consistently in for the whole file?
I started doing this already, so we have (eventual) consistency throughout the projects.

Reply:
Also, set `billed=True` right away.

Reply:
The factory doesnt seem to be setting billed if I set it in the parameters, do you have any clue as why that could be?

Reply:
Have you tried to `rm -rf .pytest_cache`?

Reply:
Looks good now.
Just FYI, you could use `report_factory(foo=bar)` instead of `report_factory.create(..)`.